src/Writer/Writer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+namespace Quidco\DbSampler\Writer;
+
+use Doctrine\DBAL\Connection;
+
+class Writer
+{
+    /**
+     * Commands to run on the destination table after importing
+     *
+     * @var array
+     */
+    protected $postImportSql = [];
+
+    /**
+     * @var Connection
+     */
+    private $destination;
+
+    public function __construct(\stdClass $config, Connection $destination)
+    {
+        $this->destination = $destination;
+
+        $this->postImportSql = isset($config->postImportSql) ? $config->postImportSql : [];
+    }
+
+    public function write(string $tableName, $row): void
+    {
+        $this->sanitiseRowKeys($row);
+        $this->destination->insert($tableName, $row);
+    }
+
+    public function postWrite(): void
```

Comment:
i don't like this method being here, but until the `Migrator` is refactored a bit more I think it's a necessary evil